xla/service/gpu/runtime/nccl_collective_thunk.h

```diff
@@ -107,6 +107,19 @@ NcclCollectiveConfig GetNcclCollectiveConfigForMlir(
   return config;
 }
 
+// This wraps the ncclCommHandle object along with other information
+// that could be useful.
+struct NcclCommHandleWrapper {
+  NcclCommHandleWrapper(NcclApi::NcclCommHandle handle, bool is_local) :
```

Comment:
Isn't it auto generated? Definitely auto-generated by clang, but subtly different in gcc, I'd try to remove it if not needed.

Reply:
No it doesnt auto-gen the proper constructor for me though, it only creates copy, move and default constructor with no arguments. I have tried gcc and clang-17, maybe im missing some flags? gcc will definitely complain cuz it's pedantic :(. I have to keep it there.